gen3/bin/mutate-guppy-config-for-pfb-export-test.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+source "${GEN3_HOME}/gen3/lib/utils.sh"
+gen3_load "gen3/gen3setup"
+
+set -xe
+
+# script for mutating the guppy configuration on jenkins env
+# the incoming PR's guppy configuration is mutated to Jenkins environment
+
+# how to run:
+# gen3 mutate-guppy-config-for-pfb-export-test {PR} {repoName}
+
+prNumber=$1
+shift
+repoName=$1
+
+if ! shift; then
+ gen3_log_err "use: mutate-guppy-config prNumber repoName"
+ exit 1
+fi
+
+kubectl get configmap manifest-guppy -o yaml > original_guppy_config.yaml
+sed -i 's/\(.*\)"index": "\(.*\)_subject",$/\1"index": "'"${prNumber}"'.'"${repoName}"'.\2_subject",/' original_guppy_config.yaml
```

Comment:
do we still need this index in this case?

Reply:
Yeah, we need exclusive indices to be utilized by portal-guppy when the PR check mutates into some gitops-qa or cdis-manifest environment's DataDictionary / EtlMapping/ gitops.json, etc.